src/main/views/logons/template.njk

```diff
@@ -42,7 +42,12 @@
         classes: "govuk-!-width-three-quarters govuk-!-margin-top-1",
         id: "emailAddress",
         name: "emailAddress",
-        value: logonForm.emailAddress
+        value: logonForm.emailAddress,
+        type: "email",
+        spellcheck: false,
+        hint: {
+          text: "We’ll use this to search the users"
```

Comment:
Do we need hint?

Reply:
As per govuk Email example , it is best to say why we are asking for emails

Reply:
https://design-system.service.gov.uk/patterns/email-addresses/